src/htmlminifier.js

```diff
@@ -1113,6 +1113,22 @@ function minify(value, options, partialMarkup) {
           currentChars += '|';
         }
       }
+
+      if (options.removeEmptyScriptTags) {
+        if (isElementEmpty && tag === 'script') {
```

Comment:
I suggest changing this line to `        if (isElementEmpty && tag === 'script' && !attrs.src) {` so that external scripts do not get removed.